.github/workflows/deploy-dev.yml

```diff
@@ -107,6 +107,8 @@ jobs:
     needs: package
     runs-on: ubuntu-latest
     environment: development
+    outputs:
+        app_name: ${{ steps.app_name.outputs.name }}
```

Comment:
Return hier de route host met:
```yaml
route_host:
   ${{ steps.route_host.outputs.name }}
```
Die kan dan in volgende jobs gebruikt worden. `app_name` zou je niet nodig moeten hebben.